Podfile.lock

```diff
@@ -36,12 +36,12 @@ PODS:
   - "GoogleUtilities/NSData+zlib (7.5.0)"
   - GoogleUtilities/Reachability (7.5.0):
     - GoogleUtilities/Logger
-  - MiniApp/Admob8 (3.5.0):
+  - MiniApp/Admob8 (3.6.0):
     - Google-Mobile-Ads-SDK (~> 8.0)
     - MiniApp/Core
-  - MiniApp/Core (3.5.0):
+  - MiniApp/Core (3.6.0):
     - ZIPFoundation (= 0.9.12)
-  - MiniApp/UI (3.5.0):
+  - MiniApp/UI (3.6.0):
```

Comment:
I guess you need to update the podfile.lock to 3.5.0

Reply:
I purposely upgraded to 3.6.0 in advance because there is the new feature that cleans up all the previous manifest we stored in keychain until 3.5.0